src/components/treemap/index.js

```diff
@@ -59,17 +59,19 @@ export class TreeMapComponent extends events.EventEmitter {
 
 
     api.aggregate(endpoint, cube, params).then((data) => {
-      var root = {
-        children: [],
-        summary: data.summary[params.aggregates],
-        summary_fmt: Utils.numberFormat(data.summary[params.aggregates])
-      };
+      var root = {};
+      root.children = [];
+      root.summary = data.summary[params.aggregates];
+      root.currency = data.currency[params.aggregates];
+      root.summary_fmt = Utils.numberFormat(data.summary[params.aggregates]);
+      root.summary_fmt_currency = Utils.moneyFormat(root.summary_fmt, root.currency);
```

Comment:
Preserving the old attributes summary_fmt and summary. Adding new attributes 'x_currency' with formatted values with currency.